### PR TITLE
publish spark docs to /developer/spark

### PR DESCRIPTION
**Pull request by moxious**
Intent is to make the following docs appear on /developer/spark

### CODE REVIEW

File: spark.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+site:
+    title: Neo4j Connector for Apache Spark
+  
+  content:
```
Comment (Mogztter) [inline spark.yml:4]: `content`, `output`, `ui`, `urls` and `asciidoc` should not be indented

File: spark.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+site:
+    title: Neo4j Connector for Apache Spark
```
Comment (Mogztter) [inline spark.yml:2]:
```suggestion
  title: Neo4j Connector for Apache Spark
```

File: spark.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - ./extensions/remote-include/remote-include-processor
+    - ./extensions/macros/macros
+    attributes:
+      experimental: ''
+      page-component: kb
+      page-theme: kb
+      page-canonical-root: /developer/spark
+      # page-breadcrumbs:
+      #   - name: Neo4j KB
+      #     url: https://neo4j.com/developer/kb/
```
Comment (Mogztter) [inline spark.yml:29]:
```suggestion
content:
  sources:
  - url: https://github.com/neo4j-contrib/neo4j-spark-connector
    branches: ['4.0']
    start_path: 'doc/docs'
  
output:
  dir: ./build/site/developer/spark
ui:
  bundle:
    url: ui/build/ui-bundle.zip
    snapshot: true
urls:
  html_extension_style: indexify
asciidoc:
  extensions:
  - ./extensions/remote-include/remote-include-processor
  - ./extensions/macros/macros
  attributes:
    experimental: ''
    page-component: kb
    page-theme: kb
    page-canonical-root: /developer/spark
    # page-breadcrumbs:
    #   - name: Neo4j KB
    #     url: https://neo4j.com/developer/kb/
```

File: spark.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      snapshot: true
+  urls:
+    html_extension_style: indexify
+  asciidoc:
+    extensions:
+    - ./extensions/remote-include/remote-include-processor
+    - ./extensions/macros/macros
+    attributes:
+      experimental: ''
+      page-component: kb
```
Comment (Mogztter) [inline spark.yml:24]:
```suggestion
```

File: spark.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      page-theme: kb
```
Comment (Mogztter) [inline spark.yml:25]:
```suggestion
      page-theme: spark
```

**Comment by Mogztter** [comment]
@moxious Could you please add a task in `package.json`:

```json
"build:spark": "antora --fetch --stacktrace spark.yml",
```

**Comment by moxious** [comment]
Apologies @Mogztter I was working with @adam-cowley in the background on slack and I think this PR was a wrong-headed approach on my part, which he helped me clarify with commits already made to the repo.  Accordingly I am going to close this.

A ticket where I'm tracking all of the documentation related stuff for this is here:  https://github.com/neo4j-contrib/neo4j-spark-connector/issues/189

**Comment by Mogztter** [comment]
Ok, no worries :wink:

**Pull request closed without merging**